packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -299,40 +299,18 @@ class FlutterPreSubmitFileComparator extends FlutterGoldenFileComparator {
     }
 
     goldens ??= SkiaGoldClient(baseDirectory);
-    await goldens.getExpectations();
-
+    await goldens.auth();
+    await goldens.tryjobInit();
     return FlutterPreSubmitFileComparator(baseDirectory.uri, goldens);
   }
 
   @override
   Future<bool> compare(Uint8List imageBytes, Uri golden) async {
     golden = _addPrefix(golden);
-    final String testName = skiaClient.cleanTestName(golden.path);
-    final List<String> testExpectations = skiaClient.expectations[testName];
-
-    if (testExpectations == null) {
-      // There is no baseline for this test
-      return true;
-    }
```

Comment:
@kjlubick, what result will be returned if the tryjob does not have a baseline?

Reply:
Oh, I forgot. I am guessing it will return false, and then be triaged.

Reply:
Yeah, that should be correct